nifi-registry-framework/src/main/java/org/apache/nifi/registry/security/authorization/file/FileAccessPolicyProvider.java

```diff
@@ -517,28 +522,22 @@ private void populateInitialAdmin(final Authorizations authorizations) {
         }
     }
 
-//    /**
-//     * Creates a user for each node and gives the nodes write permission to /proxy.
-//     *
-//     * @param authorizations the overall authorizations
-//     */
-//    private void populateNodes(Authorizations authorizations) {
-//        for (String nodeIdentity : nodeIdentities) {
-//            final User node = userGroupProvider.getUserByIdentity(nodeIdentity);
-//            if (node == null) {
-//                throw new AuthorizerCreationException("Unable to locate node " + nodeIdentity + " to seed policies.");
-//            }
-//
-//            // grant access to the proxy resource
-//            addUserToAccessPolicy(authorizations, ResourceType.Proxy.getValue(), node.getIdentifier(), WRITE_CODE);
-//
-//            // grant the user read/write access data of the root group
-//            if (rootGroupId != null) {
-//                addUserToAccessPolicy(authorizations, ResourceType.Data.getValue() + ResourceType.ProcessGroup.getValue() + "/" + rootGroupId, node.getIdentifier(), READ_CODE);
-//                addUserToAccessPolicy(authorizations, ResourceType.Data.getValue() + ResourceType.ProcessGroup.getValue() + "/" + rootGroupId, node.getIdentifier(), WRITE_CODE);
-//            }
-//        }
-//    }
+    /**
+     * Creates a user for each NiFi client and gives each one write permission to /proxy.
+     *
+     * @param authorizations the overall authorizations
+     */
+    private void populateNodes(Authorizations authorizations) {
```

Comment:
Looks like this needs to get called from the load method similar to populateInitialAdmin

Reply:
Good catch, will update